README.md

```diff
@@ -31,7 +31,17 @@ Please remember to provide a good summary, description as well as steps to repro
 | [KeycloakClient](./deploy/crds/keycloak.org_keycloakclients_crd.yaml) | Represents a client in a keycloak server                 |
 | [KeycloakBackup](./deploy/crds/keycloak.org_keycloakbackups_crd.yaml) | Manage Keycloak database backups                         |
 
-## Deploying to a Cluster
+
+## Deployment to a Kubernetes or Openshift cluster
+
+The official documentation contains installation instruction for this Operator.
+
+[Getting started with keycloak-operator](https://www.keycloak.org/getting-started/getting-started-operator-kubernetes)
```

Comment:
This link is probably wrong as it points to the Kubernetes deployment. We also have a quickstart for the OpenShift installation: https://www.keycloak.org/getting-started/getting-started-operator-openshift

Reply:
So should there be a link to the OCP and one to the k8s operator or do you think we should only link to here: https://www.keycloak.org/getting-started

Reply:
Perhaps two links would be better.

Reply:
Could you please update this? We're close to getting this merged.

Reply:
done. I guess the pipeline failing is unrelated to my changes.